distributed/tests/test_steal.py

```diff
@@ -401,9 +399,6 @@ async def test_work_steal_no_kwargs(c, s, a, b):
 
 @gen_cluster(client=True, nthreads=[("127.0.0.1", 1), ("127.0.0.1", 2)])
 async def test_dont_steal_worker_restrictions(c, s, a, b):
-    future = c.submit(slowinc, 1, delay=0.10, workers=a.address)
```

Comment:
It looks like this was used to initialize the task duration. Does removing this affect what we test?